src/main/resources/i18n/messages.properties

```diff
@@ -26,17 +24,14 @@ DbgPacker.2=Zip selected files
 DbgPacker.3=Open zip location
 DbgPacker.4=Overwrite existing file?
 Dialog.Close=Close
-Dialog.Confirm=Confirm
```

Comment:
@ler0y I don't think this one should be removed even if it's not currently used, because it's general and can be handy for future dialog buttons